### ISSUE TITLE
Additional video tutorials

### ISSUE THREAD

**Issue by rozyczko** [easyScience/easyDiffractionApp#39]
Extend the existing simulation tutorial
Add a new fitting tutorial

**Comment by AndrewSazonov** [comment]
Related easyDiffractionApp PR: https://github.com/easyScience/easyDiffractionApp/pull/45
Related easyAppGui PR: https://github.com/easyScience/easyAppGui/pull/8

**Comment by celinedurniak** [comment]
I reviewed the following tutorials: App Interface, Data simulation, Data fitting

- The stop tutorial button is very useful
- The learning outcomes could be displayed at the beginning of the tutorials
- The title of the tutorial being watched could be displayed in the stop button or in the bar at the bottom
- One could display an indication of the duration of the tutorial and / or of its progress while the tutorial is running.
- The videos are going too fast at times; making it difficult to read all displayed tips. A Pause button would be useful or giving the option to move forward / backward between stages could be another option.
- Some tutorials modify the settings at the beginning. But this might result in having no information displayed during the video (when, for example, unticking "enable tool tips").
- For Tutorial 3, the user has to remember to enable the tool tips and the user guides before starting the tutorial
- At the end of the tutorials, the settings and the tab are not reset to the user’s choice: dark/light mode, not going back to the “home” tab

I noticed 2 typos in UserGuides.qml:
- Line 34 
These tab buttons **allows** you to easily navigate between the application pages
-> These tab buttons **allow** you to easily navigate between the application pages

- Line 186

This tab button allows to see the\n**calcualted** data as plain text. ->
This tab button allows to see the\n**calculated** data as plain text

**Comment by AndrewSazonov** [comment]
Thanks for the review! The typos have been corrected. I'm not sure that all suggestions can be implemented, but in any case it will take quite some time. So I suggest creating another task to expand the existing functionality.
